packages/primitives/package.json

```diff
@@ -13,8 +13,7 @@
     "babel-plugin-emotion": "^10.0.0-beta.2"
   },
   "peerDependencies": {
-    "prop-types": "15.x",
-    "react": "15.x || 16.x",
+    "react": ">= 16.3.0",
```

Comment:
This peerDep should be removed since `@emotion/primitives-core` which is a dependency of this package has react as a peerDep